### DISCUSSION TITLE
Welcome to PSXPackager Discussions!

### DISCUSSION THREAD

**Discussion by RupertAvery** [RupertAvery/PSXPackager#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bagelbyheart** [comment]
Would this be a good place to settle a bet so to speak? Specifically, there's a lot of information out there indicating that pbp files are lossy compression. This doesn't fit my own testing at all however, with iso files that've been through the tool showing the same md5.

So, does storing your games in pbp files make any change to the games content as far as emulators are concerned?

**Reply by RupertAvery** [reply]
It depends on how much the bet is and what side you are on.

PBP files pack 1 or more CD tracks into one file. 

There is the DATA track which can contain game executable code and assets (images, audio samples, fmvs) These CANNOT be externally lossy compressed, as the _game_ does not know about external compression.  It expects them to exist as they "originally" did, when the PBP blocks are uncompressed into memory by the emulator.

Then there are AUDIO tracks which usually contain UNCOMPRESSED CD digital audio, again, as they would originally exist on a real disc. These CAN be lossy compressed (I think using ATRAC3) as the emulators are doing the decompression and playing of the CD Audio outside of the game code (the game just requests the PS1 hardware to play a track)

PSXPackager does not support audio track compression, so if you used a good dump of a game to create a PBP, you will get the same data back out.

**Reply by bagelbyheart** [reply]
Thank you, this is pretty much what I was expecting. Just to try and put a final nail in this coffin, are you aware of any pbp tools that do support audio track compression?

**Comment by yukisota105** [comment]
sorry but how to i use?I'm a bit dumb and I didn't understand very well, the executable just opens a cmd and then it disappears and doesn't do anything

**Reply by RupertAvery** [reply]
Try the GUI version

https://github.com/RupertAvery/PSXPackager/releases/download/v1.6.3/PsxPackagerGUI.zip

**Comment by v8venturi** [comment]
Made an account just to say thanks. This is a really cool bit of kit, and the GUI is epic!

**Reply by RupertAvery** [reply]
Thanks, I appreciate it!
